tests/test_playbooks/module_defaults.yml

```diff
@@ -0,0 +1,26 @@
+---
+- hosts: localhost
+  collections:
+    - theforeman.foreman
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+    - name: fetch status
+      include_tasks: tasks/status_info.yml
+
+- hosts: tests
+  collections:
+    - theforeman.foreman
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  module_defaults:
+    group/theforeman.foreman.foreman:
+      username: "{{ foreman_username }}"
+      password: "{{ foreman_password }}"
+      server_url: "{{ foreman_server_url }}"
+  tasks:
+    - name: fetch status
+      status_info:
+        validate_certs: "{{ foreman_validate_certs }}"
```

Comment:
:screwdriver: Now this works? I thought `collections:` and `module_defaults:` did not work together.

Are you quite sure, the status module _needs_ authentication? Can we also add some other `_info` module?

Reply:
https://github.com/ansible/ansible/issues/75279#issuecomment-881946053

Reply:
I did add a FQCN test and another module, just to be like suuuuuuuuper sure ;)